HACKING.md

```diff
@@ -19,19 +19,21 @@ recommended to do that:
  - It avoids having to install development packages on your main machine.
  - It avoids having to map the build and test dependencies to package names of various distributions.
 
-On Fedora/CentOS/RHEL based distributions, install toolbox with
+1. Install `toolbox`
+   
+   - Fedora/CentOS/RHEL based distributions:
 
     sudo dnf install toolbox
```

Comment:
You have to indent this further to make it a code block, as the "4 indenting spaces" is relative to the parent environment (the itemization). See https://github.com/garrett/cockpit/blob/hacking-steps/HACKING.md , it looks bad that way.

Reply:
Yeah, that's what I thought (and I see it here). I usually use code block fencing to not have to count spaces. :sweat_smile: